Rocket.Chat/Controllers/Chat/ChatViewController.swift

```diff
@@ -1003,3 +1024,16 @@ extension ChatViewController {
     }
 
 }
+
+// MARK: Alerter
+extension ChatViewController: Alerter {
```

Comment:
Why don't we add the `Alerter` in the initial declaration of the controller? I think it's more clear.